pipeline/merger/src/main/scala/weco/pipeline/merger/services/Merger.scala

```diff
@@ -115,13 +116,13 @@ trait Merger extends MergerLogging {
               state = result.mergedTarget.state,
               redirectSources = result.mergedTarget.redirectSources ++ redirectedIdentifiers
             )
-
           MergerOutcome(
-            resultWorks = redirects.toList ++ remaining ++ deleted :+ targetWork,
+            resultWorks = redirects.toList ++ passThroughTransform(
+              (remaining ++ deleted).toSeq :+ targetWork),
```

Comment:
Is this the best place to be running the passThroughTransform? Wouldn't it make more sense to run this *before* we put the works into the merger rules? Here you could end up mapping away the work that the others redirect to, no?

e.g. you have `redirects = Seq(A)` and `targetWork = B`, so `A` redirects to `B`. If you mess this up, you'll get `A` redirecting to `B` and `B` invisible, which is wrong.